src/python/pants/backend/python/dependency_inference/rules_test.py

```diff
@@ -101,21 +101,9 @@ def run_dep_inference(address: Address) -> InferredDependencies:
                 Params(InferPythonDependencies(target[PythonSources]), options_bootstrapper),
             )
 
-        # NB: We do not infer `src/python/app.py`, even though it's used by `src/python/f2.py`,
-        # because it is part of the requested address.
-        normal_address = Address("src/python")
-        assert run_dep_inference(normal_address) == InferredDependencies(
-            [
-                Address("3rdparty/python", target_name="Django"),
-                Address("src/python/util", relative_file_path="dep.py", target_name="util"),
-            ]
-        )
```

Comment:
Why remove this test case?

Reply:
I think that these were "fixes" from before the `Targets` vs `UnexpandedTargets` fix. Reverted.